tests/integration/test_chat.py

```diff
@@ -1,14 +1,12 @@
 import os
```

Comment:
Running `make isort` on Python 2.7 here yields:
```
diff --git a/tests/integration/test_chat.py b/tests/integration/test_chat.py
index 75fe128..d3b7d06 100644
--- a/tests/integration/test_chat.py
+++ b/tests/integration/test_chat.py
@@ -3,14 +3,15 @@ import re
 
 import pkg_resources
 import yaml
-from bok_choy.promise import EmptyPromise
 from ddt import data, ddt
 from django.test.client import Client
-from mock import ANY, patch
 from xblock.reference.user_service import XBlockUser
 from xblockutils.resources import ResourceLoader
 from xblockutils.studio_editable_test import StudioEditableBaseTest
 
+from bok_choy.promise import EmptyPromise
+from mock import ANY, patch
+
 loader = ResourceLoader(__name__)
```

Reply:
The above seems incorrect. I don't see why those lines should move. 

Given that one solutions is upgraded to Juniper there is no need for a python2 version, we can stick to the Python3 path.